Editor/AGS.Editor/Panes/Room/RoomEditFilters/CharactersEditorFilter.cs

```diff
@@ -222,6 +222,7 @@ public void Paint(Graphics graphics, RoomEditorState state)
 
                 if (_movingCharacterWithMouse || _movingCharacterWithKeyboard)
                 {
+                    Brush brush = new SolidBrush(Color.FromArgb(200, Color.Black));

```

Comment:
Can this be renamed `shadeBrush`? Just because it’s created a bit far from its usage.